src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -151,6 +153,11 @@ public void sortFilteredPersonList(String order) {
         addressBook.sortPersons(order);
     }
 
+    @Override
+    public ObservableList<Person> getPersonList() {
+        return persons;
```

Comment:
Perhaps you can `return addressBook.getPersonList()` instead so it removes the need of a new field called persons

Reply:
tru